index.css

```diff
@@ -0,0 +1,14 @@
+#image {
+    display: block;
+      
+    max-width: 100%;
+    
+    margin-left: auto;
+    
+    margin-right: auto;
+    
+}
```

Comment:
Please add appropriate spacing between each line. Make sure to avoid this linter error. It makes your file look messy & inconsistent